lib/ui/homePage(temporary).dart

```diff
@@ -48,16 +48,28 @@ class _MyHomePageState extends State<MyHomePage> {
     return Scaffold(
       key: _scaffoldKey,
       appBar: AppBar(
-        leading: Center(child: Text(widget.title,style: GoogleFonts.assistant(color: Colors.white,fontWeight: FontWeight.bold,fontSize: MediaQuery.of(context).size.height*0.030),)),
+        leading: Center(
+            child: Text(
+          widget.title,
+          style: GoogleFonts.assistant(
+              color: Colors.white,
+              fontWeight: FontWeight.bold,
+              fontSize: MediaQuery.of(context).size.height * 0.030),
+        )),
         actions: [
           FlatButton(
-              child: Text('Sign Out',style: GoogleFonts.assistant(color: Colors.white,fontWeight: FontWeight.bold,fontSize: MediaQuery.of(context).size.height*0.035),),
-              onPressed: () {
-                var googleSignMe =
-                    GoogleSignMeIn(whatToDo: 0, context: context);
-                googleSignMe.check();
-              },
-              ),
+            child: Text(
+              'Sign Out',
+              style: GoogleFonts.assistant(
+                  color: Colors.white,
+                  fontWeight: FontWeight.bold,
+                  fontSize: MediaQuery.of(context).size.height * 0.035),
+            ),
+            onPressed: () {
+              var googleSignMe = GoogleSignMeIn(whatToDo: 0, context: context);
```

Comment:
use the `logout()` function directly

Reply:
Also, remove things like `check()` function and `whatToDo` as they won't be used anymore.